src/main/java/systems/composable/dropwizard/cassandra/connect/EC2MultiRegionAddressTranslatorFactory.java

```diff
@@ -0,0 +1,14 @@
+package systems.composable.dropwizard.cassandra.connect;
+
+import com.datastax.driver.core.policies.AddressTranslator;
+import com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/** A factory for configuring and building {@link com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator} instances. */
+@JsonTypeName("ec2MultiRegionAddressTranslator")
```

Comment:
Please rename to `ec2MultiRegion` (to match naming formats in use for other factories)... the `AddressTranslator` part is redundant.